gyp/pylib/gyp/xcodeproj_file.py

```diff
@@ -324,8 +341,7 @@ def Copy(self):
           that._properties[key] = new_value
         else:
           that._properties[key] = value
-      elif isinstance(value, str) or isinstance(value, unicode) or \
-           isinstance(value, int):
+      elif isinstance(value, (basestring, int)):
```

Comment:
I was worried that it might break if `basestring` is a tuple, but I just tested in Python 3.6 and it works 👍